server_utils/origin.go

```diff
@@ -758,11 +784,29 @@ func CheckOriginSentinelLocations(exports []OriginExport) (ok bool, err error) {
 			if path.Base(sentinelPath) != sentinelPath {
 				return false, errors.Errorf("invalid SentinelLocation path for StoragePrefix %s, file must not contain a directory. Got %s", export.StoragePrefix, export.SentinelLocation)
 			}
-			fullPath := filepath.Join(export.StoragePrefix, sentinelPath)
-			_, err := os.Stat(fullPath)
+
+			fullPath := filepath.Join(export.FederationPrefix, sentinelPath)
+			tkn, err := generateFileTestScitoken(sentinelPath)
```

Comment:
Do you know if it would work to generate a token scoped specifically to the sentinel object instead of a token that could be used to read anything from the namespace?

Reply:
I believe the scitoken generation code is already specifically scoped to the sentinel object, does this log message look right for that?
```
time="2024-12-05T17:11:11Z" level=info msg="241205 17:11:11 8650 scitokens_Access: New valid token mapped_username=, subject=origin, issuer=https://a6c42d6578d1:33709, authorizations=/test/mock_sentinel:read,dir,stat;/pelican/monitoring/mock_sentinel:read,dir,stat" daemon=xrootd.origin
```

Reply:
Ah, you're right then, I saw that we weren't passing the "full path", but forgot we shouldn't be including the federation prefix in the token scope. Resolved!